scripts/build_relayer.sh

```diff
@@ -22,15 +22,22 @@ version_lt() {
     fi
 }
 
-# Root directory
+
+# Relayer directory
 RELAYER_PATH=$(
     cd "$(dirname "${BASH_SOURCE[0]}")"
+    cd ../relayer && pwd
+)
+
+# Base directory
+BASE_PATH=$(
+    cd $RELAYER_PATH 
```

Comment:
This seems like a backwards way of doing this. We should define `BASE_PATH` as 
`cd "$(dirname "${BASH_SOURCE[0]}")" && pwd`
and then `RELAYER_PATH` as `"$BASE_PATH"/relayer`

Reply:
Fixed in 07e43af8f585bacf6016d6b34a2a52a0d4945533